tech-cent/momo_bank_web
# #170670247 Make deposits and withdraws.

**What does this PR do?**
This PR adds functionality to enable users to signup (register) with momo bank 

**Description of the task to be completed?**
- create a home page
- create a transactions modal
- implement money deposit and withdraws

**How should this be manually tested?**
- Clone and set up the project using the instruction in the README.
- $ git checkout to `ft-make-transactions-170670247` branch which contains the changes.
- Use $ npm start to start the application
- Login to the momo bank web app
- Make a deposit or withdraw of money
You should receive feedback that the deposit or withdraw was successful

**What are the relevant pivotal tracker stories?**
[#170670247](https://www.pivotaltracker.com/story/show/170670247)

**Screenshots**
N/A
